spec/controllers/chargeback_controller_spec.rb

```diff
@@ -110,4 +111,29 @@
       expect(flash_array.first[:message]).to include("rate is assigned and cannot be deleted")
     end
   end
+
+  context "#get_cis_all" do
+    elem_objects = [:storage, :miq_enterprise, :ext_management_system, :ems_cluster, :tenant]
+    elem_objects.each do |obj|
+      let!(obj) { FactoryGirl.create(obj) }
+    end
+    elements_miq = %w(enterprise storage ext_management_system ems_cluster tenant)
+    elements_miq.each do |element|
+      classtype =
+        if element == "enterprise"
+          MiqEnterprise
+        else
+          element.classify.constantize
+        end
```

Comment:
I don't know that we need to use this clever loop over all of the types.  I was thinking of writing test for the normal case (any non-enterprise element such as storage) and the abnormal case (enterprise).  If we add a new type that is the normal case, do we need to add a new test case?  I think that's probably not necessary.  What do you think @aljesusg @lpichler?  I think the test would be easier to understand without the indirection through the loop variables.